tests/e2e/util.go

```diff
@@ -2348,6 +2350,91 @@ func deleteFcdWithRetriesForSpecificErr(ctx context.Context, fcdID string, dsRef
 	return waitErr
 }
 
+// NewDeploymentwithVolume returns a deployment spec with the specified argument.
+func NewDeploymentwithVolume(deploymentName string, replicas int32, podLabels map[string]string, imageName, image string, pvclaims []*v1.PersistentVolumeClaim, strategyType appsv1.DeploymentStrategyType) *appsv1.Deployment {
```

Comment:
See if `createDeployment()` is sufficient instead of the three methods we have added here

Reply:
yes I have utilised your `createDeployment()` method which is added recently